k8s/nginx/production/global-config.yaml

```diff
@@ -4,30 +4,38 @@ metadata:
   name: nginx-config
   namespace: nginx-ingress
 data:
-  proxy-body-size: "200m"
+  proxy-body-size:  "200m"
   client-max-body-size: "200m"
   location-snippets: "
     auth_request /auth;
     error_page 500 = @internal_server_error;
     error_page 404 = @page_not_found;
     error_page 403 = @access_denied;
     error_page 401 = @unauthorized;"
-  server-snippets: '
-    location @internal_server_error { return 500 ''{"success":false, "message":"Something went wrong!", "error":"Internal Server Error"}''; }
-    location @page_not_found { return 404 ''{"success":false, "message":"Resource not found :(", "error":"Not Found"}''; }
-    location @access_denied { return 403 ''{"success":false, "message":"Access denied :|", "error":"Insufficient Permissions"}''; }
-    location @unauthorized { return 401 ''{"success":false, "message":"Invalid authentication", "error":"Unauthorized."}''; }
-
+  server-snippets: "
+    location @internal_server_error { return 500 '{\"success\":false, \"message\":\"Something went wrong!\", \"error\":\"Internal Server Error\"}'; }
+    location @page_not_found { return 404 '{\"success\":false, \"message\":\"Resource not found :(\", \"error\":\"Not Found\"}'; }
+    location @access_denied { return 403 '{\"success\":false, \"message\":\"Access denied :|\", \"error\":\"Insufficient Permissions\"}'; }
+    location @unauthorized { return 401 '{\"success\":false, \"message\":\"Invalid authentication\", \"error\":\"Unauthorized.\"}'; }
+    
     location = /auth {
-    if ($request_uri ~ ''(/api/v1/users/loginUser)|(/api/v2/users/loginUser)'') { return 200; }
-    if ($request_uri ~ ''(/api/v1/users/forgotPassword)|(/api/v2/users/forgotPassword)'') { return 200; }
-    if ($request_uri ~ ''(/api/v1/users/candidates/register)|(/api/v2/users/candidates/register)'') { return 200; }
-    if ($request_uri !~ ''/api/v'') { return 200; } # All none api requests
+      if ($request_uri ~ '(/api/v1/users/emailAuth)|(/api/v2/users/emailAuth)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/loginUser)|(/api/v2/users/loginUser)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/emailLogin)|(/api/v2/users/emailLogin)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/forgotPassword)|(/api/v2/users/forgotPassword)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/candidates/register)|(/api/v2/users/candidates/register)') { return 200; }
+      if ($request_uri ~ '/airflow/api/v') { return 200; }    
```

Comment:
Hi @123MwanjeMike, return 200 for all airflow requests. It has its own well designed authentication mechanism.
i.e `if ($request_uri ~ '/airflow') { return 200; }`

Reply:
https://github.com/airqo-platform/AirQo-api/pull/1011#discussion_r1000264887